api/app/jobs/viirs_snow.py

```diff
@@ -271,11 +305,28 @@ async def _run_viirs_snow(self):
                         # If a different HTTPError occurred re-raise and let the next exception handler send a notification to RocketChat.
                         raise http_error
                 next_date = next_date + timedelta(days=1)
+                end = datetime.now()
+                print(f"Snow time: {end - start}")
```

Comment:
Could use logger here

Reply:
Oops, that was just benchmarking to get an idea of how long this would take. I'll remove it. Thanks!